tests/periph_spi/main.c

```diff
@@ -242,21 +241,14 @@ int cmd_transfer(int argc, char **argv)
     /* do the actual data transfer */
     spi_acquire(spi_dev);
     gpio_clear(spi_cs);
-    res = spi_transfer_bytes(spi_dev, hello, buffer, strlen(hello));
+    spi_transfer_bytes(spi_dev, hello, buffer, strlen(hello));
```

Comment:
you can replace all of the `strlen(hello)` calls by `sizeof(hello)` if you change the definition of `hello` into `char hello[] = "Hello";`